### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Character Count for Content Input

### DISCUSSION THREAD

**Discussion by vishxly** [dailydotdev/daily#1531] [Feature Requests]
### Concise Description

Display a character count while the user types content, with a limit of 4000 characters.

### Description

Currently, daily.dev allows users to submit content with a maximum length of 4000 characters, but there is no visible character counter when adding the content. This leads to difficulty in knowing how close the user is to the limit, potentially causing them to overshoot or truncate their input unintentionally. I would like to request the addition of a character counter that displays the current number of characters entered, updating in real-time as the user types. This feature would enhance the usability and user experience by making it easier to manage content length.

### Solution Proposed

- Implement a real-time character count display next to or below the content input field .
- The counter should update dynamically as users type, showing the current character count out of 4000 (e.g., "Characters: 1200/4000").
- If the user exceeds the 4000-character limit, the counter could turn red, and the system should prevent further input until the user edits the content to fit within the limit.
- Ideally, this feature should be non-intrusive and clearly visible without distracting from the writing process.

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @vishxly,

Thanks for the suggestion

**Comment by rebelchris** [comment]
Hi @vishxly ,

Thanks for the suggestion, we listened and added this counter in the bottom right.
Also updated it to 10.000 recently.

![Screenshot 2024-12-03 at 08 23 12](https://github.com/user-attachments/assets/3b925e13-ca88-4534-bba1-1c5978916de1)
